src/main.rs

```diff
@@ -1,64 +1,84 @@
 use std::io::{Stdin, Stdout, Write};
 fn main() {
-    writeln!(std::io::stdout(), "👏 Seja bem vindo(a)!!! 👏").unwrap();
+    writeln!(std::io::stdout(), "👏 Seja bem vindo(a)!!! 👏").ok();
```

Comment:
Esse `.ok()` irá ignorar o erro se for um `Result::Err`, já que converte o erro para `Option::None` (veja na doc).

Você provavelmente quis usar o `unwrap` aqui, para explodir a aplicação no caso de erro.